docs/metrics/work-in-progress.md

```diff
@@ -0,0 +1,20 @@
+# Work in progress metrics
+
+## Work in progress
+
+To provide you with an high-level view of the current work in progress of your teams, Pulse groups pull requests that are currently open by the following phases:
+
+-   **Open:** the pull request is open but the team hasn't interacted or reviewed it.
+-   **In review:** the pull request received at least one review but it isn't approved yet. Reviews include approvals, change requests, and inline pull request comments, but don't include conversation comments.
```

Comment:
```suggestion
-   **In review:** the pull request received at least one review but it isn't approved yet. These reviews include change requests and inline pull request comments, but don't include conversation comments.
```

I think saying in the first sentence that it does not include approves and then mentioning it in the second seems weird.

Reply:
Indeed, well caught! :detective: :+1: